front/pages/w/[wId]/subscription/index.tsx

```diff
@@ -63,6 +63,36 @@ export default function Subscription({
 }: InferGetServerSidePropsType<typeof getServerSideProps>) {
   const sendNotification = useContext(SendNotificationsContext);
   const [isProcessing, setIsProcessing] = React.useState<boolean>(false);
+  const router = useRouter();
+
+  useEffect(() => {
+    if (router.query.type === "cancelled") {
+      sendNotification({
+        type: "error",
+        title: "Subscribtion cancelled",
+        description: "Subscription checkout was cancelled.",
+      });
+    } else if (router.query.type === "succeeded") {
+      if (subscription.plan.code === router.query.plan_code) {
+        sendNotification({
+          type: "success",
+          title: "Subscribtion created",
+          description:
+            "You have successfully subscribed to a new plan. You can now use all the features of the new plan.",
+        });
+      } else {
+        // If the Stripe webhook is not yet received, we try waiting for it and reload the page.
+        setTimeout(() => {
```

Comment:
Let's add a notification saying we're processing  + reload the page automatically maybe every 5s ?

Reply:
We can even invite them to reload the page since the planCode is in the URL the same notification will show up right?

Reply:
I added a commit! If still processing we display a loader instead of the Plan chip, and reload every 5 seconds until it's fixed.